packages/now-node/src/helpers.ts

```
@@ -148,7 +151,8 @@ function setLazyProp<T>(req: NowRequest, prop: string, getter: () => T) {
   const opts = { configurable: true, enumerable: true };
   const optsReset = { ...opts, writable: true };
 
-  Object.defineProperty(req, prop, {
+  // @ts-ignore
```

Let's be explicit with explaining which TS error is being ignored like the docs mention here: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html

---

👍 I updated.